src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,21 @@
 import React from 'react';
+import { CommentInfo } from '../CommentInfo';
+import { Comment } from '../../types/Comment';
+import './CommentList.scss';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  comments: Comment[]
+};
+
+export const CommentList: React.FC<Props> = ({ comments }) => (
+  <div className="CommentList">
+    {comments.map(comm => (
```

Comment:
Consider using a more descriptive variable name instead of 'comm', such as 'comment'.